tools/src/main/java/org/apache/kafka/tools/TopicCommand.java

```diff
@@ -962,6 +966,10 @@ public void checkArgs() {
             if (actions != 1)
                 CommandLineUtils.printUsageAndExit(parser, "Command must include exactly one action: --list, --describe, --create, --alter or --delete");
 
+            if (has(deleteConfigOpt)) {
+                System.err.println("delete-config option has been deprecated since version 4.0 and will be fully removed in future releases.");
```

Comment:
nit: is below more aligned the message defined above as well `deleteConfigOpt = parser.accepts("delete-config", "This option is no longer supported and has been deprecated since 4.0")`
```suggestion
                System.err.println("delete-config option is no longer supported and deprecated since version 4.0. The config will be fully removed in future releases.");
```

Reply:
Thanks for suggestion.
I will apply it. 😸